.drone.yml

```diff
@@ -114,6 +114,26 @@ steps:
       exclude:
       - pull_request
 
+- name: publish-force
```

Comment:
Remove the change to the `.drone.yml` and I'll happily merge this

Reply:
Done